### PR TITLE
Lps-157721: Change Notification link to use always use '/manage' as layout friendlyURL

### PR DESCRIPTION
**Pull request by kristofloneck**
If comments needs to removed/added, just tell me and removed/add them from the code

### CODE REVIEW

**Comment by kristofloneck** [comment]
ci:test

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by seltong** [comment]
Hi @kristofloneck , could you please check the SF faults?

**Comment by seltong** [comment]
ci:test:sf

File: portal-impl/src/com/liferay/portlet/internal/PortletURLFactoryImpl.java [hunk 1]
```diff
@@ -206,6 +206,31 @@ public LiferayPortletURL create(
 
 		return _create(portletRequest, portletId, layout, lifecycle, null);
 	}
+	/*
+	Tried to minimalizing overwriting existing code as much as possible.
+	 */
+	@Override
+	public LiferayPortletURL create(
+		HttpServletRequest httpServletRequest, String portletId,
+		String lifecycle, boolean isItFromNotification) {
```
Comment (seltong) [inline portal-impl/src/com/liferay/portlet/internal/PortletURLFactoryImpl.java]: Hi @kristofloneck could you please rename `isItFromNotification` parameter to `notification`?

File: portal-impl/src/com/liferay/portlet/internal/PortletURLFactoryImpl.java [hunk 2]
```diff
@@ -224,6 +249,16 @@ else if (PortletRequest.RENDER_PHASE.equals(lifecycle)) {
 			httpServletRequest, portlet, layout, lifecycle, copy);
 	}
 
+	private LiferayPortletURL _create(
+		HttpServletRequest httpServletRequest, Portlet portlet, Layout layout,
+		String lifecycle, MimeResponse.Copy copy, boolean isItNotificaitonPortlet) {
```
Comment (seltong) [inline portal-impl/src/com/liferay/portlet/internal/PortletURLFactoryImpl.java]: Hi @kristofloneck could you please rename `isItNotificaitonPortlet` parameter to `notification`?

**Review by seltong — CHANGES_REQUESTED** [review]

**Comment by seltong** [comment]
Hi @kristofloneck sorry for delay
Could you check if it's possible fix this bug changing the `serviceContext` passed as parameter in `getNotificationLink` method? 
I will take a look at this too and try to help you as soon as possible

**Comment by MarinhoFeliphe** [comment]
Hi @kristofloneck,

I closed this pull request, feel free to reopen when you can make the mentioned changes.

Thank you,
Feliphe

**Comment by kristofloneck** [comment]
> Hi @kristofloneck sorry for delay Could you check if it's possible fix this bug changing the `serviceContext` passed as parameter in `getNotificationLink` method? I will take a look at this too and try to help you as soon as possible


Hi @seltong 
Sorry for not answering, I was quite bussy.
It is totally possible to change the layout of original request's theme display's. The main problem is we technicly lying about what is the current layout. So we need to revert request layout to the original one in the end of the `getNotificationLink`.
This was my little magic which I mentioned in the PTR.

**Pull request closed without merging**